docs/cisco.ios.ios_logging_module.rst

```diff
@@ -543,7 +543,7 @@ Status
 ------
 
 
-- This module will be removed in a release after 2023-06-01. *[deprecated]*
+- This module will be removed in version . *[deprecated]*
```

Comment:
Looks like this PR is reverting changes made by @Qalthos's PR. You'd need to fetch latest, rebase and run the add_docs again.